strategy_aws.go

```diff
@@ -0,0 +1,83 @@
+package main
```

Comment:
very minor:  I learned from Andrey that `_` is special in Go file names (which is why tests have to be in `_test.go`.  Basically `_foo` means it'll be included if the `foo` build flag is set.  ever since then I've tried to avoid underscores in filenames.  https://stackoverflow.com/a/39625565

Reply:
oof that's sneaky.

Reply:
Changed for dashes